logging/base/clusterlogforwarders/instance.yaml

```diff
@@ -4,11 +4,15 @@ metadata:
   name: instance
   namespace: openshift-logging
 spec:
```

Comment:
Let's revert the changes to instance.yaml because we use patches to override these values for the prod and test cluster to send logs remotely.